CredentialsManager/Manager.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections.Generic;
+using CredentialsManager.FilesExceptions;
+using CredentialsManager.Models;
+
+namespace CredentialsManager
+{
+  public static class Manager
+  {
+    private const string FieldDelimiter = ",";
+    private const string CredentialsFile = "Users.txt";
+    private static List<Credentials> _credentials = new();
+
+    public static void Initialize()
+    {
+      _credentials = GetCredentials();
+      _credentials.Sort();
```

Comment:
I was going to ask "why sort" but now I see that it is necessary for your efficient `CheckForDuplicates`.  In my own personal code this might be a place where I'd put an explanatory comment.  Something like `// Sorting here so CheckForDuplicates works properly`.  Or you could put the sort inside `CheckForDuplicates`.  However, it is not obvious why it is here.

Reply:
I updated this in commit [c269d01](https://github.com/csinn/CSharp-From-Zero-To-Hero-v2/pull/43/commits/c269d01381883751f5cdbbfb4ac2b879f6034f76)